course/page/code.py

```diff
@@ -137,8 +146,11 @@ def debug_print(s):
                     ["NetworkSettings"]["Ports"]["%d/tcp" % RUNPY_PORT])
             port_host_ip = port_info.get("HostIp")
 
-            if port_host_ip != "0.0.0.0":
-                connect_host_ip = port_host_ip
+            if platform.system().lower().startswith("linux"):
+                if port_host_ip != "0.0.0.0":
+                    connect_host_ip = port_host_ip
+            else:
+                connect_host_ip = getattr(settings, "RELATE_DOCKER_HOST_IP")
```

Comment:
*   I'd rather have the variable be named `RELATE_DOCKER_HOST`.
*   I'd like the variable to be used if it's defined. (not just "if it's not LInux"--that seems hard to debug and document if something goes wrong with that setting)